src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -18,6 +27,31 @@ let package = Package(
         .package(url: "https://github.com/getsentry/sentry-cocoa", from: "7.31.3"),
     ],
     targets: [
+        .target(name: "OutlineShared"),
+        .target(
+            name: "OutlineLauncher",
+            dependencies:
+                ["CocoaLumberjack",
+                 .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack"),
+                 "Tun2socks",
```

Comment:
How did you make Tun2socks work?
The existing framework didn't support mac Catalyst. You'd need to use the apple_future, but you can't depend on both.
We need one XCFramework for all platforms, but you need to build mac Catalyst Framework separately from iOS, because they target different iOS versions (13 and 11)

Reply:
/cc @jyyi1